src/pudl/workspace/datastore.py

```diff
@@ -50,15 +50,15 @@ class Datastore:
     def __init__(self, pudl_in, loglevel="WARNING", verbose=False,
                  sandbox=False, timeout=7):
         """
-        Datastore manages file retrieval for PUDL datasets.
+        Datastore `manages` file retrieval for PUDL datasets.
```

Comment:
?

Reply:
No idea wtf is gonig on there.

Reply:
should we note `manages` this? manages seems fine unless this is actually referring to something.